xla/tests/hlo_test_base.h

```diff
@@ -148,8 +148,10 @@ class HloTestBase : public ::testing::Test {
 
   // Compiles the given `hlo` with optimizations, and verifies that optimized
   // HLO matches the given FileCheck pattern.
-  void MatchOptimizedHlo(absl::string_view hlo, absl::string_view pattern,
-                         bool print_operand_shape = false);
+  void MatchOptimizedHlo(
```

Comment:
Can you please update the comment to reflect the extra argument added?

Reply:
Done.